src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -60,4 +56,65 @@ alias_netmaskbit6_tooltip : T('Select the prefix length used on the network.'),
 delete_placeholder6: T('Delete'),
 delete_tooltip6: T('Set to delete this alias.'),
 
+bridge_members_placeholder: T('Bridge Members'),
+bridge_members_tooltip: T(''),
+
+failover_critical_placeholder: T('Failover Critical'),
+failover_critical_tooltip: T(''),
+
+failover_group_placeholder: T('Failover Group'),
+failover_group_tooltip: T(''),
```

Comment:
From TrueNAS Guide "Failover" section:

"Combine multiple critical interfaces into a single Failover group. All interfaces in the group must fail before triggering a failover."

Reply:
@wblock @william-gr this seems like a reasonable tooltip label to me, what do you say?

Reply:
yes